listenbrainz/db/tests/test_do_not_recommend.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime, timedelta
```

Comment:
get_total_count and delete are not tested here.

Reply:
OK, delete is tested on the API level, but a test for get_total_count would still be good.